### ISSUE TITLE
scipy.lib.six import issue for Scipy 0.16.0

### ISSUE THREAD

**Issue by chenliu0831** [scikit-umfpack/scikit-umfpack#11]
Seems scipy 0.16.0 renamed `lib` to `_lib`

**Comment by rc** [comment]
I do not see any reference to it in scikit-umfpack sources - what problem did you find?

**Comment by chenliu0831** [comment]
Oh. I'm installing it from pip and in that version (`0.1`)  it's importing six from umfpack.py. Any plan to release new version?

**Comment by rc** [comment]
I see. :) Yes, a release could be made - will do it soon.

**Comment by rc** [comment]
So I have just put 0.2 on PyPi.

@cmutel, would you be so kind and update the wheels, please? (Doing `python setup.py sdist upload` removed the 0.1 wheels - I will put them back as soon as I recover my PyPi password, :])

**Comment by cmutel** [comment]
New wheels for OS X here:
https://dl.dropboxusercontent.com/u/1911208/scikit_umfpack-0.2-cp27-none-macosx_10_6_intel.whl
https://dl.dropboxusercontent.com/u/1911208/scikit_umfpack-0.2-cp34-cp34m-macosx_10_6_intel.whl

Note that they are compiled against SuiteSparse 4.2.1, which meant I commented out the config line:

    umf_info['libraries'].insert(0, 'rt')

However, they work fine in my simple tests.

I filed a ticket, and can update the wheels when new SuiteSparse is easy to compile: https://trac.macports.org/ticket/48830

**Comment by rc** [comment]
Great, thank you! It's on PyPi now.

@chenliu0831 does it work for you now?

**Comment by chenliu0831** [comment]
@rc Now I'm getting:

```
  Downloading scikit-umfpack-0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-NuZB8r/scikit-umfpack/setup.py", line 7, in <module>
        with open('README.md') as fd:
    IOError: [Errno 2] No such file or directory: 'README.md'
```

**Comment by rc** [comment]
Whoops. Will fix that. (TODO: write down release tasks...) I have removed the broken release from PyPi. I will create 0.2.1 tomorrow - I will submit a PR, so that people (you) can test it before uploading to PyPi.

**Comment by rc** [comment]
The problem is fixed in the master repo, but considering #13, I am waiting for comments.

**Comment by chenliu0831** [comment]
Awesome, Thanks a lot! Thumbs up for the speed.

**Comment by rc** [comment]
This is fixed by the 0.2.1 release.
